docs/1_how_it_works/1_1_how_it_works.md

```diff
@@ -0,0 +1,51 @@
+# How Finetuner works
+
+## Contrastive metric learning
+
+From an algorithmic perspective,
+**Finetuner** leverages a contrastive metric learning approach to improve your model.
+How does it work?
+
+### Step 1: Convert a model into an Embedding Model
+
+Finetuner interprets the backbone model architecture,
+removes the default *head*, applies *pooling* and freezes layers that do not need to be trained.
+For an image classification task (e.g. cats and dogs),
+Finetuner is going to remove the classification head (cat-dog classifier) and turn your model into an *embedding model*.
+
+This embedding model does not make predictions or outputs a probability,
+but instead outputs a feature vector to represent your data.
+
+### Step 2: Triplet construction and training on-the-fly
+
+Finetuner works on labeled data.
+It expects a `DocumentArray` consisting of `Document`s where each one contains `finetuner_label` corresponding to the class of a specific training example.
+
+During the fine-tuning, Finetuner creates Triplets `(anchor, positive, negative)` on-the-fly.
```

Comment:
I think we also support `CLIPLoss` right?

Reply:
yes, and it is still leveraging the triplets